addon/services/ajax.js

```diff
@@ -113,9 +113,9 @@ export default Ember.Service.extend({
         );
 
         if (response instanceof AjaxError) {
-          reject(response);
+          Ember.run.join(null, reject, response);
```

Comment:
`Ember.run` will create a new run loop (even if one is already started), `Ember.run.join` will join an existing run loop or create a new one if one isn't underway.